packages/ui/src/services/utils.ts

```diff
@@ -99,3 +99,14 @@ const getRandomString = (stringLength = 4): string => {
 export const generateUniqueId = () => {
   return `${getRandomString(8)}-${getRandomString(4)}-${getRandomString(4)}`
 }
+
+export const isParsablePositiveMeasure = (value: unknown) => {
```

Comment:
Will it be often case, where we need a positive measure? In my opinion will be nice to have something like `isParsableMeasure` and its dependant `isPositiveMeasure`.